testing/test/extensions/data/test_cert.pem

```diff
@@ -0,0 +1 @@
+testcert
```

Comment:
Same comment as above, please put these in a directory as we will have lots of extensions and we want to avoid getting these mixed